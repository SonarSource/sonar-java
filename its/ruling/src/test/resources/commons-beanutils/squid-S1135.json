{
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/ConstructorUtils.java':[
427,
],
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/MappedPropertyDescriptor.java':[
114,
],
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/PropertyUtilsBean.java':[
970,
],
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/converters/ArrayConverter.java':[
310,
],
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/converters/BooleanArrayConverter.java':[
201,
],
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/converters/SqlTimeConverter.java':[
74,
75,
],
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/converters/SqlTimestampConverter.java':[
74,
75,
],
}
