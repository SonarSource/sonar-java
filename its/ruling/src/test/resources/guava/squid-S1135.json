{
'com.google.guava:guava:src/com/google/common/base/Ascii.java':[
34,
],
'com.google.guava:guava:src/com/google/common/base/CaseFormat.java':[
174,
179,
],
'com.google.guava:guava:src/com/google/common/base/CharMatcher.java':[
493,
798,
],
'com.google.guava:guava:src/com/google/common/base/Converter.java':[
153,
163,
210,
],
'com.google.guava:guava:src/com/google/common/base/Predicates.java':[
51,
637,
],
'com.google.guava:guava:src/com/google/common/base/Splitter.java':[
370,
],
'com.google.guava:guava:src/com/google/common/base/Stopwatch.java':[
66,
],
'com.google.guava:guava:src/com/google/common/base/Throwables.java':[
197,
],
'com.google.guava:guava:src/com/google/common/base/Verify.java':[
160,
],
'com.google.guava:guava:src/com/google/common/cache/CacheBuilder.java':[
240,
],
'com.google.guava:guava:src/com/google/common/cache/LocalCache.java':[
152,
205,
374,
599,
611,
1795,
1870,
1978,
2675,
3469,
3519,
3954,
4052,
4070,
4212,
],
'com.google.guava:guava:src/com/google/common/collect/AbstractMapBasedMultimap.java':[
233,
1184,
],
'com.google.guava:guava:src/com/google/common/collect/ArrayTable.java':[
103,
139,
152,
262,
374,
478,
531,
],
'com.google.guava:guava:src/com/google/common/collect/BinaryTreeTraverser.java':[
40,
167,
],
'com.google.guava:guava:src/com/google/common/collect/Collections2.java':[
87,
],
'com.google.guava:guava:src/com/google/common/collect/ComputingConcurrentHashMap.java':[
350,
],
'com.google.guava:guava:src/com/google/common/collect/ConcurrentHashMultiset.java':[
84,
268,
],
'com.google.guava:guava:src/com/google/common/collect/Constraints.java':[
292,
],
'com.google.guava:guava:src/com/google/common/collect/EvictingQueue.java':[
125,
],
'com.google.guava:guava:src/com/google/common/collect/FluentIterable.java':[
291,
305,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingCollection.java':[
50,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingList.java':[
57,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingMap.java':[
61,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingMapEntry.java':[
57,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingSet.java':[
51,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingSortedMap.java':[
58,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableBiMap.java':[
195,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableCollection.java':[
160,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableList.java':[
225,
639,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableListMultimap.java':[
246,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableMap.java':[
238,
274,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableMultiset.java':[
51,
407,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableSet.java':[
361,
362,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableSetMultimap.java':[
256,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableSortedAsList.java':[
51,
55,
76,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableTable.java':[
47,
],
'com.google.guava:guava:src/com/google/common/collect/Iterables.java':[
785,
853,
1033,
],
'com.google.guava:guava:src/com/google/common/collect/Lists.java':[
121,
929,
],
'com.google.guava:guava:src/com/google/common/collect/MapConstraints.java':[
350,
],
'com.google.guava:guava:src/com/google/common/collect/MapMakerInternalMap.java':[
128,
177,
291,
507,
519,
1812,
1882,
1987,
],
'com.google.guava:guava:src/com/google/common/collect/Maps.java':[
2165,
2203,
],
'com.google.guava:guava:src/com/google/common/collect/MinMaxPriorityQueue.java':[
89,
157,
235,
471,
677,
],
'com.google.guava:guava:src/com/google/common/collect/Multimaps.java':[
2058,
],
'com.google.guava:guava:src/com/google/common/collect/Multiset.java':[
268,
301,
],
'com.google.guava:guava:src/com/google/common/collect/Multisets.java':[
412,
476,
521,
603,
755,
],
'com.google.guava:guava:src/com/google/common/collect/NaturalOrdering.java':[
27,
],
'com.google.guava:guava:src/com/google/common/collect/ObjectArrays.java':[
134,
186,
],
'com.google.guava:guava:src/com/google/common/collect/Ordering.java':[
104,
265,
773,
774,
910,
],
'com.google.guava:guava:src/com/google/common/collect/RegularImmutableList.java':[
110,
],
'com.google.guava:guava:src/com/google/common/collect/RegularImmutableSortedSet.java':[
40,
86,
88,
],
'com.google.guava:guava:src/com/google/common/collect/RegularImmutableTable.java':[
174,
],
'com.google.guava:guava:src/com/google/common/collect/ReverseNaturalOrdering.java':[
28,
],
'com.google.guava:guava:src/com/google/common/collect/Sets.java':[
714,
744,
],
'com.google.guava:guava:src/com/google/common/collect/SortedLists.java':[
265,
],
'com.google.guava:guava:src/com/google/common/collect/StandardRowSortedTable.java':[
52,
],
'com.google.guava:guava:src/com/google/common/collect/StandardTable.java':[
370,
],
'com.google.guava:guava:src/com/google/common/collect/Table.java':[
59,
],
'com.google.guava:guava:src/com/google/common/collect/Tables.java':[
301,
],
'com.google.guava:guava:src/com/google/common/collect/TreeBasedTable.java':[
145,
163,
],
'com.google.guava:guava:src/com/google/common/collect/TreeMultiset.java':[
948,
],
'com.google.guava:guava:src/com/google/common/collect/TreeRangeSet.java':[
103,
157,
],
'com.google.guava:guava:src/com/google/common/collect/TreeTraverser.java':[
55,
],
'com.google.guava:guava:src/com/google/common/escape/ArrayBasedCharEscaper.java':[
160,
],
'com.google.guava:guava:src/com/google/common/escape/Escaper.java':[
62,
],
'com.google.guava:guava:src/com/google/common/escape/Escapers.java':[
57,
257,
],
'com.google.guava:guava:src/com/google/common/hash/AbstractStreamingHashFunction.java':[
76,
108,
111,
],
'com.google.guava:guava:src/com/google/common/hash/BloomFilter.java':[
305,
],
'com.google.guava:guava:src/com/google/common/hash/HashCode.java':[
77,
],
'com.google.guava:guava:src/com/google/common/hash/Hashing.java':[
412,
],
'com.google.guava:guava:src/com/google/common/hash/HashingOutputStream.java':[
41,
],
'com.google.guava:guava:src/com/google/common/hash/Murmur3_128HashFunction.java':[
44,
],
'com.google.guava:guava:src/com/google/common/hash/Murmur3_32HashFunction.java':[
104,
],
'com.google.guava:guava:src/com/google/common/hash/SipHashFunction.java':[
71,
],
'com.google.guava:guava:src/com/google/common/hash/package-info.java':[
15,
],
'com.google.guava:guava:src/com/google/common/io/BaseEncoding.java':[
134,
205,
564,
],
'com.google.guava:guava:src/com/google/common/io/ByteSource.java':[
540,
],
'com.google.guava:guava:src/com/google/common/io/CharSequenceReader.java':[
33,
],
'com.google.guava:guava:src/com/google/common/io/CharStreams.java':[
263,
],
'com.google.guava:guava:src/com/google/common/math/BigIntegerMath.java':[
68,
],
'com.google.guava:guava:src/com/google/common/math/IntMath.java':[
86,
],
'com.google.guava:guava:src/com/google/common/math/LongMath.java':[
84,
126,
],
'com.google.guava:guava:src/com/google/common/net/HostSpecifier.java':[
99,
],
'com.google.guava:guava:src/com/google/common/net/InetAddresses.java':[
641,
],
'com.google.guava:guava:src/com/google/common/net/InternetDomainName.java':[
454,
],
'com.google.guava:guava:src/com/google/common/net/MediaType.java':[
100,
],
'com.google.guava:guava:src/com/google/common/net/PercentEscaper.java':[
96,
97,
],
'com.google.guava:guava:src/com/google/common/primitives/Booleans.java':[
120,
174,
],
'com.google.guava:guava:src/com/google/common/primitives/Bytes.java':[
48,
96,
150,
],
'com.google.guava:guava:src/com/google/common/primitives/Chars.java':[
153,
207,
],
'com.google.guava:guava:src/com/google/common/primitives/Doubles.java':[
75,
143,
201,
624,
],
'com.google.guava:guava:src/com/google/common/primitives/Floats.java':[
73,
139,
197,
601,
],
'com.google.guava:guava:src/com/google/common/primitives/Ints.java':[
161,
215,
],
'com.google.guava:guava:src/com/google/common/primitives/Longs.java':[
128,
182,
],
'com.google.guava:guava:src/com/google/common/primitives/Shorts.java':[
160,
214,
],
'com.google.guava:guava:src/com/google/common/primitives/SignedBytes.java':[
39,
99,
],
'com.google.guava:guava:src/com/google/common/primitives/UnsignedInteger.java':[
149,
],
'com.google.guava:guava:src/com/google/common/primitives/UnsignedLong.java':[
71,
],
'com.google.guava:guava:src/com/google/common/reflect/TypeResolver.java':[
414,
],
'com.google.guava:guava:src/com/google/common/reflect/TypeToken.java':[
1150,
],
'com.google.guava:guava:src/com/google/common/reflect/Types.java':[
444,
450,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/AbstractExecutionThreadService.java':[
121,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/AbstractService.java':[
175,
214,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/AtomicDouble.java':[
105,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/AtomicDoubleArray.java':[
113,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/ForwardingCheckedFuture.java':[
57,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/ForwardingFuture.java':[
73,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/ForwardingListenableFuture.java':[
51,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/Futures.java':[
1306,
1529,
1725,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/ListenableFutureTask.java':[
38,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/ListenerCallQueue.java':[
39,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/Monitor.java':[
203,
204,
206,
239,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/MoreExecutors.java':[
621,
825,
826,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/RateLimiter.java':[
92,
113,
133,
404,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/SerializingExecutor.java':[
34,
38,
41,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/SimpleTimeLimiter.java':[
118,
188,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/Uninterruptibles.java':[
262,
329,
],
}
