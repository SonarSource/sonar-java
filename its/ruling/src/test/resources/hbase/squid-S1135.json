{
'org.apache.hbase:hbase-protocol:src/main/java/com/google/protobuf/HBaseZeroCopyByteString.java':[
58,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/SnapshotProtos.java':[
1247,
1255,
1446,
1456,
2008,
2018,
2028,
2041,
],
'org.apache.hbase:hbase-protocol:src/main/java/org/apache/hadoop/hbase/protobuf/generated/WALProtos.java':[
3975,
3983,
4262,
4272,
4957,
4967,
4977,
4993,
],
}
