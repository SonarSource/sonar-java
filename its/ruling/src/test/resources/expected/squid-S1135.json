{
'project:com/sun/corba/se/impl/activation/ServerMain.java':[
45,
],
'project:com/sun/imageio/plugins/gif/GIFStreamMetadata.java':[
36,
],
'project:com/sun/imageio/plugins/png/PNGImageReader.java':[
1094,
],
'project:com/sun/imageio/plugins/png/PNGImageWriter.java':[
364,
],
'project:com/sun/imageio/plugins/png/PNGMetadata.java':[
254,
],
'project:com/sun/java/swing/plaf/windows/WindowsMenuItemUI.java':[
169,
],
'project:com/sun/java/swing/plaf/windows/XPStyle.java':[
265,
],
'project:com/sun/jmx/snmp/tasks/ThreadService.java':[
142,
193,
],
'project:com/sun/org/apache/bcel/internal/generic/ReferenceType.java':[
188,
222,
248,
253,
306,
315,
320,
],
'project:com/sun/org/apache/xalan/internal/xslt/EnvironmentCheck.java':[
559,
616,
630,
736,
902,
956,
1102,
1103,
],
'project:com/sun/org/apache/xalan/internal/xsltc/cmdline/Transform.java':[
88,
94,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/LocationPathPattern.java':[
44,
48,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Mode.java':[
577,
1426,
1455,
1477,
1502,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Number.java':[
564,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Output.java':[
51,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/RelationalExpr.java':[
243,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Stylesheet.java':[
1350,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/VariableBase.java':[
113,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/WithParam.java':[
200,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/XPathParser.java':[
2785,
2803,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/XslAttribute.java':[
161,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/MultiDOM.java':[
558,
570,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/SAXImpl.java':[
1596,
1598,
],
'project:com/sun/org/apache/xalan/internal/xsltc/runtime/AbstractTranslet.java':[
99,
],
'project:com/sun/org/apache/xerces/internal/dom/CoreDocumentImpl.java':[
88,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLScanner.java':[
298,
463,
941,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/SchemaDVFactoryImpl.java':[
49,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/regex/RegularExpression.java':[
474,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XMLSchemaLoader.java':[
946,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSConstraints.java':[
1153,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/models/CMBuilder.java':[
245,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDHandler.java':[
2739,
2759,
],
'project:com/sun/org/apache/xerces/internal/jaxp/SAXParserImpl.java':[
389,
426,
449,
524,
],
'project:com/sun/org/apache/xerces/internal/jaxp/datatype/DurationImpl.java':[
202,
346,
539,
550,
561,
564,
587,
589,
590,
591,
592,
593,
595,
629,
631,
632,
633,
635,
637,
656,
658,
659,
660,
662,
664,
1969,
],
'project:com/sun/org/apache/xerces/internal/jaxp/datatype/XMLGregorianCalendarImpl.java':[
313,
1121,
],
'project:com/sun/org/apache/xerces/internal/jaxp/validation/ValidatorHandlerImpl.java':[
896,
897,
908,
],
'project:com/sun/org/apache/xerces/internal/util/NamespaceContextWrapper.java':[
69,
],
'project:com/sun/org/apache/xerces/internal/util/ParserConfigurationSettings.java':[
292,
],
'project:com/sun/org/apache/xerces/internal/util/SecurityManager.java':[
179,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XIncludeHandler.java':[
1441,
1443,
1828,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XIncludeMessageFormatter.java':[
29,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XPointerFramework.java':[
69,
],
'project:com/sun/org/apache/xml/internal/dtm/DTMFilter.java':[
165,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDefaultBase.java':[
1773,
2063,
2071,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDocumentImpl.java':[
1707,
1899,
1907,
1943,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMManagerDefault.java':[
149,
518,
798,
815,
830,
846,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/dom2dtm/DOM2DTM.java':[
1393,
1401,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/sax2dtm/SAX2DTM.java':[
706,
1068,
1076,
1226,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/sax2dtm/SAX2DTM2.java':[
56,
],
'project:com/sun/org/apache/xml/internal/security/Init.java':[
271,
],
'project:com/sun/org/apache/xml/internal/security/c14n/CanonicalizerSpi.java':[
42,
],
'project:com/sun/org/apache/xml/internal/security/c14n/implementations/Canonicalizer20010315.java':[
331,
347,
],
'project:com/sun/org/apache/xml/internal/security/c14n/implementations/NameSpaceSymbTable.java':[
256,
407,
],
'project:com/sun/org/apache/xml/internal/security/encryption/XMLCipher.java':[
1070,
1264,
2164,
2168,
2183,
2369,
2383,
2471,
2538,
2598,
2601,
3735,
3736,
3936,
],
'project:com/sun/org/apache/xml/internal/security/keys/KeyInfo.java':[
413,
594,
1075,
],
'project:com/sun/org/apache/xml/internal/security/keys/content/PGPData.java':[
31,
],
'project:com/sun/org/apache/xml/internal/security/keys/content/SPKIData.java':[
31,
],
'project:com/sun/org/apache/xml/internal/security/keys/content/X509Data.java':[
419,
],
'project:com/sun/org/apache/xml/internal/security/keys/content/x509/XMLX509Certificate.java':[
154,
],
'project:com/sun/org/apache/xml/internal/security/keys/keyresolver/KeyResolver.java':[
334,
349,
],
'project:com/sun/org/apache/xml/internal/security/signature/ObjectContainer.java':[
37,
],
'project:com/sun/org/apache/xml/internal/security/signature/Reference.java':[
163,
],
'project:com/sun/org/apache/xml/internal/security/signature/XMLSignatureInput.java':[
53,
588,
],
'project:com/sun/org/apache/xml/internal/security/transforms/implementations/FuncHere.java':[
146,
],
'project:com/sun/org/apache/xml/internal/security/transforms/implementations/TransformXPath2Filter.java':[
220,
],
'project:com/sun/org/apache/xml/internal/security/transforms/params/XPath2FilterContainer.java':[
281,
],
'project:com/sun/org/apache/xml/internal/security/transforms/params/XPath2FilterContainer04.java':[
242,
],
'project:com/sun/org/apache/xml/internal/security/transforms/params/XPathFilterCHGPContainer.java':[
261,
],
'project:com/sun/org/apache/xml/internal/security/utils/Base64.java':[
215,
],
'project:com/sun/org/apache/xml/internal/security/utils/Constants.java':[
211,
],
'project:com/sun/org/apache/xml/internal/security/utils/resolver/implementations/ResolverDirectHTTP.java':[
59,
105,
],
'project:com/sun/org/apache/xml/internal/serialize/DOMSerializerImpl.java':[
84,
],
'project:com/sun/org/apache/xml/internal/serializer/OutputPropertiesFactory.java':[
318,
],
'project:com/sun/org/apache/xml/internal/serializer/SerializerBase.java':[
250,
],
'project:com/sun/org/apache/xml/internal/serializer/ToStream.java':[
906,
2364,
2892,
3293,
3311,
3323,
3343,
],
'project:com/sun/org/apache/xml/internal/serializer/TreeWalker.java':[
263,
271,
],
'project:com/sun/org/apache/xml/internal/serializer/WriterToUTF8Buffered.java':[
236,
365,
],
'project:com/sun/org/apache/xml/internal/serializer/utils/DOM2Helper.java':[
119,
],
'project:com/sun/org/apache/xml/internal/utils/DOM2Helper.java':[
222,
299,
],
'project:com/sun/org/apache/xml/internal/utils/DOMHelper.java':[
63,
190,
415,
544,
608,
900,
935,
1075,
1101,
1109,
],
'project:com/sun/org/apache/xml/internal/utils/ListingErrorHandler.java':[
301,
372,
381,
],
'project:com/sun/org/apache/xml/internal/utils/QName.java':[
386,
],
'project:com/sun/org/apache/xpath/internal/NodeSet.java':[
342,
],
'project:com/sun/org/apache/xpath/internal/NodeSetDTM.java':[
498,
],
'project:com/sun/org/apache/xpath/internal/SourceTreeManager.java':[
143,
166,
204,
382,
],
'project:com/sun/org/apache/xpath/internal/axes/AxesWalker.java':[
448,
],
'project:com/sun/org/apache/xpath/internal/axes/FilterExprIteratorSimple.java':[
131,
],
'project:com/sun/org/apache/xpath/internal/axes/LocPathIterator.java':[
609,
1022,
],
'project:com/sun/org/apache/xpath/internal/axes/MatchPatternIterator.java':[
321,
],
'project:com/sun/org/apache/xpath/internal/axes/PredicatedNodeTest.java':[
259,
491,
],
'project:com/sun/org/apache/xpath/internal/axes/UnionChildIterator.java':[
142,
],
'project:com/sun/org/apache/xpath/internal/axes/UnionPathIterator.java':[
334,
],
'project:com/sun/org/apache/xpath/internal/domapi/XPathResultImpl.java':[
152,
],
'project:com/sun/org/apache/xpath/internal/patterns/StepPattern.java':[
506,
612,
],
'project:java/awt/GraphicsDevice.java':[
558,
],
'project:java/awt/event/MouseEvent.java':[
976,
],
'project:java/awt/image/ComponentSampleModel.java':[
155,
218,
],
'project:java/awt/peer/ComponentPeer.java':[
91,
101,
109,
278,
390,
406,
],
'project:java/awt/peer/FramePeer.java':[
111,
124,
],
'project:java/lang/Integer.java':[
309,
],
'project:java/security/CodeSource.java':[
636,
],
'project:java/text/PatternEntry.java':[
241,
],
'project:java/text/RBCollationTables.java':[
276,
],
'project:java/text/RBTableBuilder.java':[
191,
],
'project:java/util/JapaneseImperialCalendar.java':[
828,
1306,
1338,
],
'project:java/util/SimpleTimeZone.java':[
438,
698,
705,
],
'project:java/util/concurrent/ConcurrentLinkedDeque.java':[
500,
726,
],
'project:java/util/concurrent/LinkedBlockingDeque.java':[
834,
],
'project:javax/management/openmbean/OpenMBeanAttributeInfoSupport.java':[
816,
817,
832,
833,
],
'project:javax/management/openmbean/OpenMBeanParameterInfoSupport.java':[
397,
398,
413,
414,
],
'project:javax/sound/midi/Track.java':[
66,
],
'project:javax/sound/sampled/AudioSystem.java':[
47,
],
'project:javax/swing/plaf/basic/BasicScrollBarUI.java':[
214,
],
'project:javax/swing/plaf/nimbus/AbstractRegionPainter.java':[
633,
],
'project:javax/swing/plaf/nimbus/NimbusDefaults.java':[
1563,
],
'project:javax/swing/plaf/nimbus/SynthPainterImpl.java':[
2766,
],
'project:javax/swing/plaf/synth/SynthTabbedPaneUI.java':[
308,
320,
569,
],
'project:javax/swing/text/TabStop.java':[
136,
],
'project:javax/swing/text/html/HTMLEditorKit.java':[
1877,
1907,
1933,
2086,
],
'project:javax/swing/text/rtf/RTFAttribute.java':[
57,
],
'project:javax/swing/text/rtf/RTFAttributes.java':[
213,
409,
],
'project:javax/swing/text/rtf/RTFGenerator.java':[
125,
144,
163,
308,
333,
662,
741,
796,
853,
905,
951,
960,
988,
],
'project:javax/swing/text/rtf/RTFParser.java':[
114,
234,
319,
],
'project:javax/swing/text/rtf/RTFReader.java':[
125,
138,
214,
704,
1226,
1232,
1242,
1310,
1330,
1359,
1616,
1630,
],
'project:javax/xml/bind/ContextFinder.java':[
269,
368,
371,
],
'project:javax/xml/bind/JAXBContext.java':[
443,
470,
],
'project:javax/xml/bind/helpers/ValidationEventLocatorImpl.java':[
149,
],
'project:javax/xml/datatype/Duration.java':[
979,
],
'project:javax/xml/transform/stax/StAXSource.java':[
105,
],
}
